synapseclient/table.py

```diff
@@ -1134,13 +1144,14 @@ def build_table(name, parent, values):
     except:
         pandas_available = False
 
-    if not isinstance(values, pd.DataFrame):
+    if not isinstance(values, pd.DataFrame) and not isinstance(values, six.string_types):
         raise ValueError("Values of type %s is not yet supported." % type(values))
     if not pandas_available:
         raise ValueError("pandas package is required.")
     cols = as_table_columns(values)
     schema = Schema(name=name, columns=cols, parent=parent)
-    return Table(schema, values)
+    headers = [SelectColumn.from_column(col) for col in cols]
```

Comment:
Per @zimingd, we should refactor the name of SelectColumn to something else that represents what the class is: type information for each column of the CSV file. SelectColumn used to be used for only for CSVs retrieved from synapse; in that case, SelectColumn represented the types of columns that were in the SQL SELECT clause (including aggregates).